.travis.yml

```diff
@@ -0,0 +1,49 @@
+os:
+  - linux
+#  - osx
+
+language: cpp
+
+addons:
+  apt:
+    packages:
+    - libqt4-dev
+    - libvtk5-dev
+    - libvtk5-qt4-dev
+    - python-vtk
+    - python-numpy
+
+notifications:
+  email: false
+
+branches:
+  only:
+    - master
+
+before_install:
+  - bash ./distro/travis/before_install.sh
```

Comment:
@patmarion I remembered why I broke off the drake pre-build contents into their own script: Travis can't handle multi-line bash IF statements inside `.travis.yml`: https://github.com/travis-ci/travis-ci/issues/497 https://groups.google.com/forum/#!topic/travis-ci/uaAP9zEdiCg 

But putting them inside a `.sh` file and then calling that script works just fine.